test/backend-cpp/ModelBuilder.cpp

```diff
@@ -63,8 +63,8 @@ bool ModelBuilder::runAndVerifyTest(std::vector<OMTensorUniquePtr> &inputs,
   assert(!inputs.empty() && "Expecting valid inputs");
 
   // Run the test code.
-  onnx_mlir::ExecutionSession execSession(
-      getSharedLibName(sharedLibBaseName), "run_main_graph");
+  onnx_mlir::ExecutionSession execSession(getSharedLibName(sharedLibBaseName));
+  execSession.setEntryPoint("run_main_graph");
```

Comment:
Can you confirm that this call is redundant as the exec session uses by default the "run_main_graph" name.

Reply:
Yes, it's redundant. Done.

Reply:
Its fine if you wanted to test the functionality, but if you do this, just add a comment as otherwise folks may think that it is still needed.